### ISSUE TITLE
Inconsistency in handling query parameters for list and search endpoints

### ISSUE THREAD

**Issue by timeu** [Arabidopsis-Information-Portal/adama#95]
Query parameters are handled differently in the `search` and in the `list` endpoint. 

A query parameter in the `search` endpoint is treated as a normal `key,value` pair ` "limit": 10`, however a query parameter in the `list` endpoint is always handled as a `key,value[]` pair `"limit":[10]`. 

Futhermore the `list` endpoint can handle arrays of parameters: 
`my_service_v_0.1?limit=10,20` --> `"limit":[10,20]` whereas the `search`  endpoint always only takes the first parameter: `"limit":10`. 

BTW both endpoints have the same parameter definition in the `metadata.yml` (example for `search` endpoint):

```yml
endpoints:
    /search:
        parameters:
            - name: LIMIT
              description: How many records should be retrieved
              type: number
              required: false
              format: integer
```

I didn't find anything in the documentation about this.

**Comment by waltermoreira** [comment]
Confirmed.  Fixed in commit 357155b0bdb50e11a05e5d9555833de3dedf1801

**Comment by waltermoreira** [comment]
Forgot to comment, the handling of pair of values is now consistent too.  But note that the proper way is to pass either an array `[10,20]` or to pass twice the argument `limit`: `limit=10&limit=20`.  In either case, it should be declared as an array in the `/endpoints`.
